osbuild/loop.py

```diff
@@ -182,21 +191,11 @@ def flush_buf(self) -> None:
         linux.ioctl_blockdev_flushbuf(self.fd)
 
     def set_fd(self, fd):
-        """Bind a file descriptor to the loopback device
-
-        The loopback device must be unbound. The backing file must be
-        either a regular file or a block device. If the backing file is
-        itself a loopback device, then a cycle must not be created. If
-        the backing file is opened read-only, then the resulting
-        loopback device will be read-only too.
-
-        Parameters
-        ----------
-        fd : int
-            the file descriptor to bind
+        """
+        Deprecated, use configure instead.
```

Comment:
Please add "TODO" comment to delete this eventually once image-info used by osbuild CI stops using this method. If pylint screams, just set it to ignore this one instance of TODO.